### PR TITLE
Don't run auto-triage for closed issues

### PR DESCRIPTION
**Pull request by cbjeukendrup**

### CODE REVIEW

File: .github/workflows/ci_issues_triage.yml [hunk 1]
```diff
@@ -52,7 +55,10 @@ jobs:
         labeled: VST
 
   assign:
-    if: github.event.action == 'labeled' || github.event.action == 'unlabeled'
+    if: |
+      (github.event.action == 'labeled' || github.event.action == 'unlabeled')
+      && github.event.issue.state == 'open'
+      && !contains( github.event.issue.labels.*.name, 'disable-auto-assign')
```
Comment (Eism) [inline .github/workflows/ci_issues_triage.yml:61]: please explain how it works

Comment (cbjeukendrup) [reply .github/workflows/ci_issues_triage.yml:61]: Sometimes, you want to add a label but not automatically assign someone. The idea is that you can first add a new 'disable-auto-assign' label, then add the label you want to add, and finally remove the 'disable-auto-assign' label. 

This line just checks whether the 'disable-auto-assign' label is present (and whether the issue is currently open).

Comment (Eism) [reply .github/workflows/ci_issues_triage.yml:61]: Can you give examples of issues where this caused problems?
It just seems to me that this is a workaround and not a real solution

Comment (cbjeukendrup) [reply .github/workflows/ci_issues_triage.yml:61]: It's not necessarily a solution or workaround, but actually a feature. For example, it is useful for old issues: often, you do want to give them the correct label, but you don't want to disturb people by automatically assigning them. 
Or, as @oktophonie mentioned, sometimes for closed issues you do want to add the label (just for the record), but not assign people and also not make the issue automatically appear in a project.

So, most of the times, you do want "auto-assign", but sometimes you don't want it. This PR adds the possibility to do that.

If you have a better idea for how we can disable "auto-assign" for single cases, that's welcome though.

Comment (Eism) [reply .github/workflows/ci_issues_triage.yml:61]: I agree with not auto-assigning to closed issues. It is not obvious in what cases we need to disable auto-assign for opened issues

Comment (cbjeukendrup) [reply .github/workflows/ci_issues_triage.yml]: Okay, I removed it. Then we simply have to accept the fact that we'll sometimes be annoyed with "musebot assigned you" notifications for old issues when someone is adding missing labels.

**Review by Eism — APPROVED** [review]

**Pull request merged**